### ISSUE TITLE
If you steal another persons work then at least try to hide xd

### ISSUE THREAD

**Issue by mAa4a97** [revinewyd/SpotifyPremium#3]
That is just lazy and delete "your" repository scum.
![image](https://github.com/revinewyd/SpotifyPremium/assets/34139457/d2c292f8-949e-4e4e-ae80-0eae1761d89d)

**Comment by JJorne-JDEVJ** [comment]
oohh that get my blood boiling, anyways here's the original : https://github.com/mrpond/BlockTheSpot

**Comment by abhinav4848** [comment]
Why does this one show up on DuckDuckGo instead of the real one.

**Comment by H1Mohamed** [comment]
> Why does this one show up on DuckDuckGo instead of the real one.

yeah for some reason it appears as the first in DuckDuckGo's search results while the original doesn't even show up in the first page, but thanks for the warning @mAa4a97
